docs/topics/settings.rst

```diff
@@ -1765,8 +1745,7 @@ Type of priority queue used by the scheduler. Another available type is
 ``scrapy.pqueues.DownloaderAwarePriorityQueue``.
 ``scrapy.pqueues.DownloaderAwarePriorityQueue`` works better than
 ``scrapy.pqueues.ScrapyPriorityQueue`` when you crawl many different
-domains in parallel. But currently ``scrapy.pqueues.DownloaderAwarePriorityQueue``
-does not work together with :setting:`CONCURRENT_REQUESTS_PER_IP`.
```

Comment:
I think the lack of support for this setting was the only reason we did not make `DownloaderAwarePriorityQueue` the default priority queue back in the day (@wRAR @kmike Do you recall?). Nothing needs to happen in this PR about it, but I wonder if we should merge `DownloaderAwarePriorityQueue` into `ScrapyPriorityQueue` now or, if there is a reason to keep a simpler `ScrapyPriorityQueue` around, if we should make `DownloaderAwarePriorityQueue` default, at least with `startproject`.

Reply:
I remember basically nothing about `DownloaderAwarePriorityQueue` merging 😅

Reply:
The initial message of #3520 indeed says that.

If that's still the only reason I'm fine with changing the default, not sure about removing `ScrapyPriorityQueue` while `CONCURRENT_REQUESTS_PER_IP` is still supported.

Reply:
Separate.